java/src/main/java/ai/rapids/cudf/HostColumnVectorCore.java

```diff
@@ -371,6 +372,23 @@ public final boolean getBoolean(long index) {
     return offHeap.data.getBoolean(index * type.getSizeInBytes());
   }
 
```

Comment:
I would like to see APIs that let us get the unscaled value INT/LONG from the decimal instead of creating an instance of BigDecimal each time.  We could either update DType to indicate that DECIMAL32 `isBackedByInt` and DECIMAL64 `isBackedByLong` along with some documentation changes or new APIs that make ti clear what you are getting.  Either is fine with me.

Reply:
I applied the former approach. Currently, DECIMAL32 `isBackedByInt` and DECIMAL64 `isBackedByLong`. And related test cases have also been updated.